Game/Content/Spatial/PartitionMetadata/LegacyMetadata.schema

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+package gdk.loadbalancing;
+
+import "improbable/standard_library.schema";
+
+component VirtualWorkerAssignment {
+     id = 190500;
```

Comment:
I might've missed this - did we increase `SpatialConstants::STARTING_GENERATED_COMPONENT_ID` to prevent generated components from overlapping with these component IDs?

Reply:
Fair question.

Reply:
Changed to land into the declared external component range